### PR TITLE
Show key messages

### PR DESCRIPTION
**Pull request by moonkii**
Create `KeyMessage` component.

Add key messages into locales text.

Screenshot

### CODE REVIEW

**Comment by moonkii** [comment]
key message

File: src/components/KeyMessage.jsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    [mq(640)]: {
+      marginLeft: 0,
+    },
+  },
+  strong: {
+    position: 'relative',
+    display: 'inline-block',
+    margin: '0 .1em .1em',
+    fontWeight: 'bold',
+    '& span:first-of-type': {
```
Comment (ahastudio) [inline src/components/KeyMessage.jsx]: https://developer.mozilla.org/docs/Web/CSS/:first-of-type

File: src/components/KeyMessage.jsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      zIndex: 1,
+      position: 'absolute',
+      top: '.1em',
+      bottom: 0,
+      left: '-.1em',
+      right: '-.1em',
+      background: '#3dfafd',
+    },
+    '& span:last-of-type': {
```
Comment (ahastudio) [inline src/components/KeyMessage.jsx]: https://developer.mozilla.org/docs/Web/CSS/:last-of-type

File: src/components/KeyMessage.jsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+            {line.split('*')
+              .map((part, index) => (
+                index % 2 === 0 ? part : (
+                  <strong
+                    key={part}
+                    css={styles.strong}
+                  >
+                    <span />
+                    <span>{part}</span>
+                  </strong>
```
Comment (ahastudio) [inline src/components/KeyMessage.jsx]: 나중에 이건 `StrongText` 같은 컴포넌트로 분리하면 좋겠네요.

**Pull request merged**